ctags.lua

```diff
@@ -220,12 +220,32 @@ local function tselect_cmd(tag, force)
 		vis:info(string.format('Tag not found: %s', tag))
 	else
 		local keys = {}
-		for i = 1, #matches do
-			table.insert(keys, matches[i].desc)
+
+		-- find max width for pretty printing
+		local maxdescwidth = 0
+		for _, match in ipairs(matches) do
+			if match.desc:len() > maxdescwidth then
+				maxdescwidth = match.desc:len()
+			end
+		end
+		local fmt = '%-'..maxdescwidth..'s %s'
+		for _, match in ipairs(matches) do
+			table.insert(keys, fmt:format(match.desc, match.excmd))
```

Comment:
I do not not why, but `vis-menu` behaves very weird when the `excmd` contains a tab. Since the value should just give a general idea of how the match looks like, I would suggest replacing "%s+" with a single space.